<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>testGenerationForm</class>
 <widget class="QMainWindow" name="testGenerationForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>460</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>460</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Test Generation Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="os">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Desktop Operating Systems / Platforms</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="windowsCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Windows</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="macCheckBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mac OSX</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="linuxCheckBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Linux</string>
    </property>
   </widget>
   <widget class="QLabel" name="browser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chromeCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Chrome: 1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="firefoxCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FireFox:  1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="safariCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Safari</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="operaCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Opera</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="defaultBrowserCheckBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Default (Mobile)</string>
    </property>
   </widget>
   <widget class="QSlider" name="chromeSlider">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="firefoxSlider">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
    <property name="tickPosition">
     <enum>QSlider::NoTicks</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="deviceBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>All</string>
    </property>
   </widget>
   <widget class="QLabel" name="devices">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Devices (mobile)</string>
    </property>
   </widget>
   <widget class="QLabel" name="Qualty">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quality: # of Browser Versions</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="testSuiteCheckBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add to Test Suite?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="testSuiteEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>390</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="testNameEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>none</string>
    </property>
   </widget>
   <widget class="QComboBox" name="suiteBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="acceptButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>420</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Accept</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cencelButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>420</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="IECheckBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Internet Explorer</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="androidEmulatedBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Android (Emulated)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="samsungGalaxyNexusBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Samsung Galaxy Nexus</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="googleNexus7CBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Google Nexus 7C</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="HTCOneXBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>HTC One X</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="motorolaDroid4Box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Motorola Droid 4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="iPhoneBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>iPhone</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="iPadBox">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>iPad</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="androidCheckBox">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Android</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="iosCheckBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>87</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>iOS</string>
    </property>
   </widget>
   <widget class="QLabel" name="suiteLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>370</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Suite Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="testLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Test Name</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
